src/App.js

```diff
@@ -186,8 +186,9 @@ class App extends Component {
         <NavigationBar />
       ) : null;
     const renderFooter =
-      (skillListRegex.test(pathname) && pathLength > 2 && pathLength <= 4) ||
-      renderFooterPagesList.includes(pathname) ? (
+      ((skillListRegex.test(pathname) && pathLength > 2 && pathLength <= 4) ||
+        renderFooterPagesList.includes(pathname)) &&
+      !window.location.pathname.includes('/admin/') ? (
```

Comment:
You don't need `window.location.pathname` just `pathname` will be enough.Change it.

Reply:
@harsh-jindal Made the changes.